include/hw_unique_key.h

```diff
@@ -22,34 +22,26 @@ extern "C" {
 
 #include <stdbool.h>
 
-#ifdef __ZEPHYR__
+#if defined(__NRF_TFM__)
```

Comment:
Should this be 
```suggestion
#if !defined(__NRF_TFM__)
```

Reply:
No, I want to include the header when building in TF-M.